[FIX] change remaining latin expressions (etc and i.e.) (#628)
bids-standard/bids-specification: src/04-modality-specific-files/05-task-events.md

| onset           | REQUIRED              | Onset (in seconds) of the event measured from the beginning of the acquisition of the first volume in the corresponding task imaging data file. If any acquired scans have been discarded before forming the imaging data file, ensure that a time of 0 corresponds to the first image stored. In other words negative numbers in "onset" are allowed<sup>5</sup>.                                                                                                     |
| duration        | REQUIRED              | Duration of the event (measured from onset) in seconds. Must always be either zero or positive. A "duration" value of zero implies that the delta function or event is so short as to be effectively modeled as an impulse.                                                                                                                                                                                                                                            |

> | sample          | OPTIONAL              | Onset of the event according to the sampling scheme of the recorded modality (that is: referring to the raw data file that the `events.tsv` file accompanies).                                                                                                                                                                                                                                                                                                            |

Reviewer: ```suggestion
| sample          | OPTIONAL              | Onset of the event according to the sampling scheme of the recorded modality (that is, referring to the raw data file that the `events.tsv` file accompanies).                                                                                                                                                                                                                                                                                                            |
```

| trial_type      | OPTIONAL              | Primary categorisation of each trial to identify them as instances of the experimental conditions. For example: for a response inhibition task, it could take on values "go" and "no-go" to refer to response initiation and response inhibition experimental conditions.                                                                                                                                                                                              |
| response_time   | OPTIONAL              | Response time measured in seconds. A negative response time can be used to represent preemptive responses and "n/a" denotes a missed response.                                                                                                                                                                                                                                                                                                                         |

> | stim_file       | OPTIONAL              | Represents the location of the stimulus file (image, video, sound, and so on) presented at the given onset time. There are no restrictions on the file formats of the stimuli files, but they should be stored in the /stimuli folder (under the root folder of the dataset; with optional subfolders). The values under the stim_file column correspond to a path relative to "/stimuli". For example "images/cat03.jpg" will be translated to "/stimuli/images/cat03.jpg". |

Reviewer: ```suggestion
| stim_file       | OPTIONAL              | Represents the location of the stimulus file (such as an image, video or audio file) presented at the given onset time. There are no restrictions on the file formats of the stimuli files, but they should be stored in the /stimuli folder (under the root folder of the dataset; with optional subfolders). The values under the stim_file column correspond to a path relative to "/stimuli". For example "images/cat03.jpg" will be translated to "/stimuli/images/cat03.jpg". |
```
